mindmeld/components/classifier.py

```diff
@@ -150,12 +149,11 @@ def __init__(self, resource_loader):
         self.config = None
         self.hash = ""
 
-    def fit(self, queries=None, label_set=None, incremental_timestamp=None, **kwargs):
```

Comment:
Keeping queries ends up adding complexity with the new lazy loading ProcessedQueryList.  Since queries could be a list that a user passes in there would need to be code to check the type before performing operations on it that might fail.

Also, the code path to load the dataset for a classifier is easier to reason about without the queries param being there.  It's always loaded from the resource_loader by the label_set.

Reply:
@KKsharma99  Is there anywhere in the AL codepath that involved directly passing in query objects to fit?
@serapio Similarly, do we have any scripts that directly call NLP functions on query objects?